samples/apps/copilot-chat-app/webapp/src/components/chat/ChatInput.tsx

```diff
@@ -168,19 +172,15 @@ export const ChatInput: React.FC<ChatInputProps> = (props) => {
                             setValue(chatInput.value);
                         }
                     }}
-                    onChange={(_event, data) => setValue(data.value)}
+                    onChange={(_event, data) => {
+                        setValue(data.value);
+                        dispatch(editConversationInput({ id: selectedId, newInput: data.value }));
```

Comment:
Might be a good idea to implement a debounce here so it's not dispatching an action on every keydown (updating the UI can be expensive). 

Declare ` const keyStrokeTimeout = useRef(-1);` before the return statement 

```suggestion
      window.clearTimeout(keyStrokeTimeout.current);
      keyStrokeTimeout.current = window.setTimeout(() => {
            dispatch(editConversationInput({ id: selectedId, newInput: data.value }));
      }, 300);
```

Reply:
Actually it doesn't matter since the conversation.input property isn't tracked by this component so it doesn't have an effect on component state / re-rendering